go.mod

```diff
@@ -327,6 +326,7 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/internal/retry v1.10.0 // indirect
 	go.opentelemetry.io/otel/metric v0.32.1 // indirect
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
+	go.uber.org/atomic v1.7.0 // indirect
```

Comment:
@zmb3 Shouldn't you also increase the go version in the `go.mod` file? We explicitly want to use atomics from 1.19.

Reply:
I think that is also why CodeQL is complaining. UInt64 was introduced in Go 1.19 and the CodeQL build is using Go 1.18
https://pkg.go.dev/sync/atomic@master#Uint64

Reply:
Good catch! Yep.